src/commands/wait.for.page.to.load.js

```diff
@@ -3,7 +3,10 @@
 module.exports = function waitForPageToLoad () {
     /**
      * Wait for page to get fully loaded
-     * @param {callback} callback - A callback to run
      */
-    return browser.waitUntil(stepsConfig.finishedLoadingConditions);
+    return new Promise(function (resolve) {
```

Comment:
May be it would be better to use `browser.pause` here ?

Reply:
Done